sympy/utilities/iterables.py

```diff
@@ -961,6 +961,40 @@ def rotate_right(x, y):
     return x[y:] + x[:y]
 
 
+def least_rotation(x):
+    '''
+    Returns the number of steps of right rotation required to
```

Comment:
You say "right" but use left in the code above. The example here is good; it would be better to add
```
>>> rotate_left(a, _)
[1, 2, 3, 1, 5]
```